src/vumps_localham.jl

```diff
@@ -120,7 +168,7 @@ function left_environment(hᴸ, 𝕙ᴸ, ψ; tol=1e-15)
   # Get the rest of the environments in the unit cell
   Hᴸ = InfiniteMPS(Vector{ITensor}(undef, N))
   Hᴸ[N] = Hᴸᴺ¹
-  Hᴸᴺ¹ = translatecell(Hᴸᴺ¹, -1)
+  #Hᴸᴺ¹ = translatecell(Hᴸᴺ¹, -1) #Found it was useless?
```

Comment:
Good point, not sure why that was there. It doesn't look like `Hᴸᴺ¹` is used again after this, probably left over from an old version of the code.